src/commands.rs

```diff
@@ -1307,6 +1307,15 @@ struct ObslogArgs {
     /// documented and is likely to change)
     #[clap(long, short = 'T')]
     template: Option<String>,
+    /// Show patch compared to the previous version of this change
+    ///
+    /// If the previous version has different parents, it will be temporarily
+    /// rebased to the parents of the new version, so the diff is not
+    /// contaminated by unrelated changes.
+    #[clap(long, short = 'p')]
+    patch: bool,
+    #[clap(flatten)]
+    diff_format: DiffFormatArgs,
```

Comment:
This may be touching into how commands are organized, but should this be a config somewhere that applies to all displayed diffs, rather than command-specific?

Reply:
I meant, can you set the format globally in `~/.jjrc` (I forgot the actual name of the jj config file :) ) so that it applies to all diffs, rather than having to specify it for every command?
(feel free to leave that for the future though)

Reply:
Ah, I see. Yes, that's implemented here: https://github.com/martinvonz/jj/blob/02346de60e43a1590a2da3b4ce0d0bcaf3f596e0/src/commands.rs#L2389-L2404